tools/repo-tool

```diff
@@ -176,12 +199,19 @@ if [ "$1" = "-b" -o "$1" = "-ru" ]; then
 
   if [ "$1" = "-b" ]; then
     [ -n "$3" ] && project="$3" || project="$(ls -1 projects)"
```

Comment:
When `-b` is used without a project and it runs for all projects - for project `RPi` I don't think `build()` will succeed without a device (as it won't be able to determine `ARCH`).